tools/docker/tests/test_flows.sh

```diff
@@ -26,7 +26,11 @@ container_ip() {
 
 container_CAPI_port() {
     # get the CAPI port based on the container name.
-    docker exec -t "$1" grep ucc_listener_port ${installdir}/config/beerocks_controller.conf  | cut -f2 -d= | cut -f1 -d' '
+    if [ "$1" = "gateway" ]; then
```

Comment:
It is slightly weird that gateway == controller because the gateway also has agents. However, indeed, for test_flows, we always consider the gateway to be a controller-only device (i.e. the agent on the gateway is not addressed by CAPI).

So I'd add a comment here to explain why gateway == controller.